### DISCUSSION TITLE
Thanks!

### DISCUSSION THREAD

**Discussion by barotto** [mikke89/RmlUi#404] [General]
I've been reading [this](https://nolanlawson.com/2017/03/05/what-it-feels-like-to-be-an-open-source-maintainer/) blog post recently and realized how almost all of GitHub's notifications are about something lacking in a project, forming a constant stream of negativity.
So I just wanted to express my gratitude for the constant support that this project receives, which is absolutely amazing.
Thank you Michael.
-Marco

**Comment by mikke89** [comment]
Wow, thanks Marco!

The post you linked to felt very familiar, although I would say at a smaller scale. There's a surprisingly amount of time that goes into pure maintenance and responding to issues and inquiries. Sometimes I just want to work on new interesting things for the library, as that is what I get the most enjoyment out of, but find that the maintenance takes time away from that. With that said, I also find it fulfilling to help out, and especially when I see people actively using the library in their projects. I think working on the library is a lot more enjoyable knowing that somebody will actually use what I make.

And luckily, mostly everyone is nice and respectful, and I feel like we have a good community. Although I find that part about people disappearing particularly familiar. I think this is somewhat the nature of open source, people come and go based on their needs, and that is understandable. Then it is also extra nice when I see the old-timers occasionally stopping by! ;)

It's very nice to hear you appreciate the effort I put into this project. Cheers!
